bazel/repositories.bzl

```diff
@@ -673,16 +673,6 @@ def _fuzzit_linux():
 def _com_github_grpc_grpc():
     _repository_impl(
         "com_github_grpc_grpc",
-        patches = [
```

Comment:
Should there be a delete of these patches as well in this PR?

Reply:
Yes, 1.25.0 contains these patches. Sorry this was an early draft just to reproduce the failure I was seeing. Fixed so I don't forget later.